javaslang/src/main/java/javaslang/Predicates.java

```diff
@@ -138,4 +138,28 @@ private Predicates() {
         Objects.requireNonNull(predicates, "predicates is null");
         return anyOf(predicates).negate();
     }
+
+    /**
+     * A combinator that checks if <strong>one or more</strong> elements satisfy the {@code predicate}.
+     *
+     * @param predicate The predicate to check
+     * @param <T>        closure over tested object types
+     * @return A new {@code Predicate}
+     */
+    public static <T> Predicate<Value<T>> exists(Predicate<? super T> predicate) {
+        Objects.requireNonNull(predicate, "predicate is null");
+        return obj -> obj.exists(predicate);
+    }
+
+    /**
+     * A combinator that checks if <strong>all</strong> elements satisfy the {@code predicate}.
+     *
+     * @param predicate The predicate to check
+     * @param <T>        closure over tested object types
+     * @return A new {@code Predicate}
+     */
+    public static <T> Predicate<Value<T>> forAll(Predicate<? super T> predicate) {
+        Objects.requireNonNull(predicate, "predicate is null");
+        return obj -> obj.forAll(predicate);
+    }
```

Comment:
I thought it is a good idea to use `Iterable` instead of `Value`. Please change your impl accordingly:

```java
    /**
     * Lifts a given {@code Predicate} to a new {@code Predicate} that tests
     * if an element of an {@code Iterable} satisfies the given {@code Predicate}.
     *
     * @param predicate A {@code Predicate} that tests elements of type {@code T}
     * @param <T> The element type
     * @return a new {@code Predicate} that tests if a given {@code Iterable} contains an element that satisfies the given {@code predicate}
     */
    public static <T> Predicate<Iterable<T>> exists(Predicate<? super T> predicate) {
        Objects.requireNonNull(predicate, "predicate is null");
        return iterable -> Iterator.ofAll(iterable).exists(predicate);
    }

    /**
     * Lifts a given {@code Predicate} to a new {@code Predicate} that tests
     * if all elements of an {@code Iterable} satisfy the given {@code Predicate}.
     *
     * @param predicate A {@code Predicate} that tests elements of type {@code T}
     * @param <T> The element type
     * @return a new {@code Predicate} that tests if a given {@code Iterable} contains an element that satisfies the given {@code predicate}
     */
    public static <T> Predicate<Iterable<T>> forAll(Predicate<? super T> predicate) {
        Objects.requireNonNull(predicate, "predicate is null");
        return iterable -> Iterator.ofAll(iterable).forAll(predicate);
    }
```

I like your introducing javadoc sentence more.